Silicon/CommonSocPkg/Include/Library/WatchDogTimerLib.h

```diff
@@ -10,7 +10,8 @@
 
 #define  WDT_TIMEOUT_TCC_DSO   200          // 200 seconds
 #define  WDT_FLAG_TCC_DSO      BIT17
-#define  WDT_FLAG_MASK         BIT17
+#define  WDT_FLAG_TCC_BAD_DSO  BIT18
```

Comment:
In this case, I feel we need to rename WDT_FLAG_TCC_DSO to be more specific so that it is clear on the difference with BAD_DSO.   Maybe we can use WDT_FLAG_TCC_DSO_IN_PROGRESS ?

Reply:
Thx for the comment. Will rename it.